ingestion/examples/workflows/powerbi.yaml

```diff
@@ -5,15 +5,19 @@ source:
     config:
       clientId: client_id
       clientSecret: client_secret
-      redirectURI: http://localhost:8585/callback
       hostPort: https://analysis.windows.net/powerbi
+      tenantId: tenant_id
       scope:
-      - scope
-      - https://analysis.windows.net/powerbi/api/App.Read.All
-      credentials: path
+        - https://analysis.windows.net/powerbi/api/.default
       type: PowerBI
   sourceConfig:
-    config: {}
+    config:
+      chartFilterPattern:
+        includes:
+          - Total Revenue
```

Comment:
isn't this an array? also this should be a regex. Putting a regex pattern would be a better example

Reply:
yes YAML takes array like these below. I'll add more elements in the array and add a regex for better example

```
chartFilterPattern:
  includes:
    - Total Revenue
    - Total Cost
    - Total Sales
```